package.json

```diff
@@ -46,11 +46,13 @@
   },
 
   "scripts": {
-	"test": "mocha tests.coffee --compilers coffee:coffee-script/register"
+    "test": "mocha test/dice_test.coffee --compilers coffee:coffee-script/register"
```

Comment:
Since the test passes you can merge this PR and then do move tests.coffee in and do that part?